<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIStuMS</class>
 <widget class="QMainWindow" name="UIStuMS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background-color: rgb(251, 255, 255);}
#page_personInfo{background-color: rgb(251, 255, 255);}
#page_personGrade{background-color: rgb(251, 255, 255);}
#page_selectedCourse{background-color: rgb(251, 255, 255);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="LeftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>111</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 96, 115);</string>
    </property>
    <widget class="QPushButton" name="btnLookPersonInfo">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>135</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	margin-right:3px;
	margin-bottom:0px;
	color: rgb(255, 255, 255);
	background-color: #00d0ef;
	border:1px outset rgb(255, 255, 255, 80%);
	border-radius:6px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 12px;
	font-weight: bold;
	font: 12px &quot;Microsoft YaHei&quot;;
} 

/*鼠标放在按钮上方*/
QPushButton:hover {
	background-color: rgba(165, 205, 255,80%);
	border:2px outset rgba(36, 36, 36,0);
} 

/*鼠标点击按钮*/
QPushButton:pressed {
	background-color: rgba(165, 205, 255,90%);
	border:4px outset rgba(36, 36, 36,0);
}</string>
     </property>
     <property name="text">
      <string>查看本人信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLookPersonGrade">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>185</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	margin-right:3px;
	margin-bottom:0px;
	color: rgb(255, 255, 255);
	background-color: #00d0ef;
	border:1px outset rgb(255, 255, 255, 80%);
	border-radius:6px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 12px;
	font-weight: bold;
	font: 12px &quot;Microsoft YaHei&quot;;
} 

/*鼠标放在按钮上方*/
QPushButton:hover {
	background-color: rgba(165, 205, 255,80%);
	border:2px outset rgba(36, 36, 36,0);
} 

/*鼠标点击按钮*/
QPushButton:pressed {
	background-color: rgba(165, 205, 255,90%);
	border:4px outset rgba(36, 36, 36,0);
}</string>
     </property>
     <property name="text">
      <string>查询本人成绩</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>485</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
	font-size: 12px;  </string>
     </property>
     <property name="text">
      <string>退出系统</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelectedCourse">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>235</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	margin-right:3px;
	margin-bottom:0px;
	color: rgb(255, 255, 255);
	background-color: #00d0ef;
	border:1px outset rgb(255, 255, 255, 80%);
	border-radius:6px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 12px;
	font-weight: bold;
	font: 12px &quot;Microsoft YaHei&quot;;
} 

/*鼠标放在按钮上方*/
QPushButton:hover {
	background-color: rgba(165, 205, 255,80%);
	border:2px outset rgba(36, 36, 36,0);
} 

/*鼠标点击按钮*/
QPushButton:pressed {
	background-color: rgba(165, 205, 255,90%);
	border:4px outset rgba(36, 36, 36,0);
}</string>
     </property>
     <property name="text">
      <string>查询已选课程</string>
     </property>
    </widget>
    <widget class="QLabel" name="SWU">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>96</width>
       <height>96</height>
      </rect>
     </property>
     <property name="text">
      <string>SWU</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>115</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>641</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_personInfo">
     <widget class="QLabel" name="psifName">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>390</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 27px &quot;华文隶书&quot;;</string>
      </property>
      <property name="text">
       <string>蔡前哲</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="psifID">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>390</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 27px &quot;华文隶书&quot;;</string>
      </property>
      <property name="text">
       <string>202100001</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="psifClass">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>350</y>
        <width>390</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 27px &quot;华文隶书&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="psifCollege">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>390</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 27px &quot;华文隶书&quot;;</string>
      </property>
      <property name="text">
       <string>计算机与信息科学学院 软件学院</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="psifGender">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>390</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 27px &quot;华文隶书&quot;;</string>
      </property>
      <property name="text">
       <string>男</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="psifMajor">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>390</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 27px &quot;华文隶书&quot;;</string>
      </property>
      <property name="text">
       <string>计算机科学与技术</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>99</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>149</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>199</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>249</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>学院：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>299</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>专业：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>349</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>班级：</string>
      </property>
     </widget>
     <widget class="QLabel" name="grid">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>79</y>
        <width>531</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px dashed #242424;
border-color: #475594;
border-radius: 20px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>grid</zorder>
     <zorder>psifName</zorder>
     <zorder>psifID</zorder>
     <zorder>psifClass</zorder>
     <zorder>psifCollege</zorder>
     <zorder>psifGender</zorder>
     <zorder>psifMajor</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
    </widget>
    <widget class="QWidget" name="page_personGrade">
     <widget class="QTableView" name="GradeView">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>51</y>
        <width>611</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView{  
    color: black;  	/*字体颜色*/
    border: 1px solid rgb(148, 107, 86);          /*边框颜色*/  
    gridline-color:rgb(255, 233, 217);     	/*grid线颜色*/  
    background-color: #fcfcfc;  	/*无数据行颜色*/
    alternate-background-color:rgba(231, 255, 233, 100); /*行交替颜色*/
    selection-color: black;   
    selection-background-color: rgb(255, 253, 225); /*选中行背景颜色*/ 
	font-size: 12px;   
}  
  
QTableView::item:!alternate:!selected{  
    background-color: white;    /*交替行的另一颜色*/
    selection-background-color: rgb(220, 220, 220);  
    selection-color: black; 
}  

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;华文楷体&quot;;</string>
      </property>
      <property name="text">
       <string>您的成绩如下：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_selectedCourse">
     <widget class="QTableView" name="CourseView">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>51</y>
        <width>611</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView{  
    color: black;  	/*字体颜色*/
    border: 1px solid rgb(148, 107, 86);          /*边框颜色*/  
    gridline-color:rgb(255, 233, 217);     	/*grid线颜色*/  
    background-color: #fcfcfc;  	/*无数据行颜色*/
    alternate-background-color:rgba(231, 255, 233, 100); /*行交替颜色*/
    selection-color: black;   
    selection-background-color: rgb(255, 253, 225); /*选中行背景颜色*/  
	font-size: 12px;  
}  
  
QTableView::item:!alternate:!selected{  
    background-color: white;    /*交替行的另一颜色*/
    selection-background-color: rgb(220, 220, 220);  
    selection-color: black; 
}  

</string>
      </property>
     </widget>
     <widget class="QLabel" name="CourseCnt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19px &quot;华文楷体&quot;;</string>
      </property>
      <property name="text">
       <string>您共选择 门课程进行学习！ 详细信息如下：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="TopWidget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="SystemTime">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	font-size: 12px;  </string>
     </property>
     <property name="text">
      <string>系统时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="WelcomSays">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WD粗钢笔行楷-凤殿优化版</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">	font-size: 16px;  </string>
     </property>
     <property name="text">
      <string>欢迎您，同学！</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>UIStuMS</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
